src/monitoring_service/handlers.py

```diff
@@ -368,9 +372,13 @@ def action_monitoring_triggered_event_handler(event: Event, context: Context) ->
     if channel.update_status:
         last_onchain_nonce = channel.update_status.nonce
 
+    user_address = monitor_request.non_closing_signer
```

Comment:
I'd move that in the call next line

Reply:
Yes that would be better, but then the line is too long. The function call is a monster!

Reply:
Can you inline this as well